bin/octokit-payloads.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const validate = require('../payload-schemas');
+
+fs.readdirSync('./payload-examples/api.github.com').forEach(filename => {
+  const [event, ...unused] = filename.split('.');
+  const fileBuffer = fs.readFileSync(`./payload-examples/api.github.com/${filename}`);
```

Comment:
[💡 suggestion]: Not sure if there would be performance improvements on this but, do you think would be possible to use asynchronous `readdir` (https://nodejs.org/api/fs.html#fs_fs_readdir_path_options_callback) or `fs.promises.readdir` (https://nodejs.org/api/fs.html#fs_fspromises_readdir_path_options)?

Reply:
Thanks for the suggestion!

I've seen only synchronous methods throughout the project and just went with it. Personally, I don't think that there would be any measurable difference between synchronous and asynchronous methods here, but I'd be OK with replacing them with asynchronous ones should you ask me to :)

Reply:
Sure! If you think it would not be an impact and the repo is using sync everywhere... to me it's fine. It was just a suggestion 😄

Reply:
for build scripts like this I prefer synchronous methods, they are easier to reason about. If this could would run inside e.g. a server request handler, then we wouldn't want to block the event loop, and the async methods should be used. But that doesn't apply here